xpti/src/xpti_proxy.cpp

```diff
@@ -74,7 +75,8 @@ class ProxyLoader {
       {XPTI_NOTIFY_SUBSCRIBERS, "xptiNotifySubscribers"},
       {XPTI_ADD_METADATA, "xptiAddMetadata"},
       {XPTI_QUERY_METADATA, "xptiQueryMetadata"},
-      {XPTI_TRACE_ENABLED, "xptiTraceEnabled"}};
+      {XPTI_TRACE_ENABLED, "xptiTraceEnabled"},
```

Comment:
Extra comment since the new XPTI API has been commented out? If a new API is added, it must be implemented in the proxy library and the dispatcher.

Reply:
thanks for the guidance, will update

Reply:
fixed. added xptiForceSetTraceEnabled. xptiTryToEnable is implemented in proxy only and is not needed to be implemented and exposed in xptifw.